packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -1513,37 +1514,93 @@ class _RenderRangeSlider extends RenderBox with RelayoutWhenSystemFontsChangeMix
     );
   }
 
-  @override
-  void describeSemanticsConfiguration(SemanticsConfiguration config) {
-    super.describeSemanticsConfiguration(config);
+  /// Describe the semantics of the start thumb.
+  SemanticsNode _startSemanticsNode = SemanticsNode();
 
-    config.isSemanticBoundary = isEnabled;
+  /// Describe the semantics of the end thumb.
+  SemanticsNode _endSemanticsNode = SemanticsNode();
+
+  // Create the semantics configuration for a single value.
+  SemanticsConfiguration _createSemanticsConfiguration(
+      double value,
+      String label,
+      VoidCallback increaseAction,
+      VoidCallback decreaseAction,
+  ) {
+    final SemanticsConfiguration config = SemanticsConfiguration();
+    config.isEnabled = isEnabled;
+    config.textDirection = textDirection;
     if (isEnabled) {
-      config.textDirection = textDirection;
-      config.customSemanticsActions = <CustomSemanticsAction, VoidCallback>{
-        _decreaseStart: _decreaseStartAction,
-        _increaseStart: _increaseStartAction,
-        _decreaseEnd: _decreaseEndAction,
-        _increaseEnd: _increaseEndAction,
-      };
-      if (semanticFormatterCallback != null) {
-        config.value = semanticFormatterCallback(_state._lerpRangeValues(values));
-      } else {
-        config.value = values.toString();
-      }
+      config.onIncrease = increaseAction;
+      config.onDecrease = decreaseAction;
+    }
+    config.label = label ?? '';
+    if (semanticFormatterCallback != null) {
+      config.value = semanticFormatterCallback(_state._lerp(value));
+      config.increasedValue = semanticFormatterCallback(_state._lerp((value + _semanticActionUnit).clamp(0.0, 1.0) as double));
+      config.decreasedValue = semanticFormatterCallback(_state._lerp((value - _semanticActionUnit).clamp(0.0, 1.0) as double));
+    } else {
+      config.value = '${(value * 100).round()}%';
```

Comment:
Should percent be used here? If so, what about localization?

Reply:
This matches what `Slider` currently does. If they are to be localized, that would have to be a separate PR